src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/RemoveAzureRmDataLakeStoreItemAcl.cs

```diff
@@ -47,40 +47,37 @@ public class RemoveAzureDataLakeStoreItemAcl : DataLakeStoreFileSystemCmdletBase
                 "Indicates that the ACL should be removed on the file with the specified ACL without prompting.")]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Indicates a boolean response should be returned indicating the result of the delete operation."
+            )]
+        public SwitchParameter PassThru { get; set; }
+
         public override void ExecuteCmdlet()
         {
-            if (!Force.IsPresent)
-            {
-                ConfirmAction(
-                    Force.IsPresent,
-                    string.Format(Resources.RemovingDataLakeStoreItemAcl,
-                        Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
-                    string.Format(Resources.RemoveDataLakeStoreItemAcl,
-                        Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
-                    Path.OriginalPath,
-                    () =>
-                    {
-                        if (Default)
-                        {
-                            DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
-                        }
-                        else
-                        {
-                            DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
-                        }
-                    });
-            }
-            else
-            {
-                if (Default)
-                {
-                    DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
-                }
-                else
+            ConfirmAction(
+                Force,
+                string.Format(Resources.RemovingDataLakeStoreItemAcl,
+                    Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
+                string.Format(Resources.RemoveDataLakeStoreItemAcl,
+                    Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
+                Path.OriginalPath,
+                () =>
                 {
-                    DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
-                }
-            }
+                    if (Default)
+                    {
+                        DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
+                    }
+                    else
+                    {
+                        DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
+                    }
+
+                    if (PassThru)
+                    {
+                        WriteObject(true);
```

Comment:
Its a good question. In this case, failure will result in an exception, so no false will be returned. If they want an actual value returned from this cmdlet (which otherwise would not return anything) the result will always be either true or exception. If there is a better way that we should be handling this I am happy to do so, though.